SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public extension Array {
-    public func ip_subArrayFromIndices(_ indices: [Int]) -> [Element] {
+    public func ip_subArrayFrom(indices: [Int]) -> [Element] {
```

Comment:
After looking at the guidelines again I agree with you now. I do think it should be `verb(prepositionAndNouns noun)` though for clarity within the method body. I'll update the PR.